packages/perseus/src/widgets/video/video.tsx

```diff
@@ -132,4 +117,6 @@ export default {
     defaultAlignment: "block",
     supportedAlignments: ["block", "float-left", "float-right", "full-width"],
     widget: Video,
+    // TODO: things that aren't interactive shouldn't need validators
+    validator: () => noopValidator(),
```

Comment:
I don't _totally_ agree, but I could be sold on this. IMO having defaults that don't do anything (1) add cognitive noise and (2) makes it harder to tell what is/isn't be used. For instance if you were trying to remove the `accessible` property, it might have made it seem more widely used if every widget export implemented it. Granted you would see that it was declared and maybe not used, but we have a lot of misdirection in both Perseus itself and across repos that makes it difficult to see what is actually being used.